test/code_size/embind_hello_wasm.json

```diff
@@ -1,10 +1,10 @@
 {
   "a.html": 552,
   "a.html.gz": 380,
-  "a.js": 9005,
```

Comment:
I mentioned this in Discord, I'm going to need some help here.

For a few tests `--rebaseline` generates wildly different sizes on my WIndows machine than those in the repo or on CI for some reason.

Reply:
You might also be interested in this Dockerfile:
https://github.com/emscripten-core/emscripten/pull/23990#issuecomment-2770475864

Reply:
Omg that's cursed. Should we maybe just use Node for gzipping or something? (anything where our version is pinned)

Reply:
That could maybe work yes, as long as it not dynamically linked against the system zlib.

Reply:
What's even more interesting, now that I'm looking at the logs deeper, it's the Wasm size that differs, not JS. I'm definitely on latest `tot`, I cleared the cache, how can binary be different :/